packages/sanity/src/field/types/datetime/diff/DatetimeFieldDiff.tsx

```diff
@@ -1,15 +1,27 @@
+import {StringSchemaType} from '@sanity/types'
+import {Box} from '@sanity/ui'
 import React from 'react'
 import {DiffFromTo} from '../../../diff'
 import {DiffComponent, StringDiff} from '../../../types'
 import {DatetimePreview} from '../preview'
 
-export const DatetimeFieldDiff: DiffComponent<StringDiff> = ({diff, schemaType}) => {
+export interface DatetimeFieldDiffProps {
+  diff: StringDiff
+  schemaType: StringSchemaType
+}
+
+export const DatetimeFieldDiff: DiffComponent<StringDiff> = ({
+  diff,
+  schemaType,
+}: DatetimeFieldDiffProps) => {
   return (
-    <DiffFromTo
-      align="center"
-      diff={diff}
-      schemaType={schemaType}
-      previewComponent={DatetimePreview}
-    />
+    <Box data-testid="datetime-field-diff">
```

Comment:
Are we adding a `div` just for the sake of testing? If so, we should avoid that.

Could we do `<DiffFromTo data-testi="datetime-field-diff" ... />` instead?